bemvindo/tour.md

```diff
@@ -1,2 +1,306 @@
+
+var occupations = [
+	"Malcolm": "Capitão"
+	"Kaylee", "Mecânico"
+]
+
+occupations["Jayne"] = "Relações Públicas"
+```
+
+Para criar uma lista ou um dicionário vazio, use a sintaxe de inicializadores:
+
+```swift
+let emptyArray = [String]()
+let emptyDictionary = [String: Float]()
+```
+
+Se a informação do tipo puder ser inferida, você pode escrever uma lista vazia como `[]` e um dicionário vazio como `[:]` - por exemplo, quando você define um novo valor para uma variável ou passa um argumento para um função.
+
+```swift
+shoppingList = []
+occupations = [:]
+```
+
+### Fluxo de Controle
+
+Utilize `if` e `switch` para criar condições e use `for-in`, `for`, `while` e `repeat-while` para criar laços de repetição. Parênteses ao redor da condição ou da variável de laço são opcionais. Chaves ao redor do bloco de código são obrigatórias.
+
+```swift
+let individualScores = [75, 43, 103, 87, 12]
+var teamScore = 0
+for score in individualScores {
+    if score > 50 {
+        teamScore += 3
+    } else {
+        teamScore += 1
+    }
+}
+print(teamScore)
+```
+
+Na declaração de um `if`, a condição deve ser uma expressão booleana - isso significa que código como `if score { ... }` é um erro, não uma comparação implícita com zero.
+
+Você pode usar `if` e `let` juntos para trabalhar com valores que podem estar ausentes. Estes valores são representados como opcionais. Um valor opcional ou contém um valor, ou contém `nil`, para indicar a ausência de valor. Escreva uma ponto de interrogação (`?`) após o tipo de um valor para marcar o valor como opcional.
+
+```swift
+var optionalString: String? = "Hello"
+print(optionalString == nil)
+
+var optionalName: String? = "John Appleseed"
+var greeting = "Hello!"
+if let name = optionalName {
+    greeting = "Hello, \(name)"
+}
+```
+
+>EXPERIMENTO
+>
+>Altere `optionalName` para `nil`. Que saudação você recebe? Adicione uma cláusula `else` que define uma saudação diferente se `optionalName` for `nil`.
+
+Se o valor opcional for `nil`, a condição é `false` e o código entre chaves é pulado. Caso contrário, o valor opcional é desembrulhado e atribuído à constante após o `let`, tornando o valor desembrulhado disponível dentro do bloco de código.
+
+Outra maneira de lidar com valores opcionais é fornecer um valor padrão usando o operador `??`. Se o valor opcional estiver ausente, o valor padrão será utilizado.
+
+```swift
+let nickName: String? = nil
+let fullName: String = "John Appleseed"
+let informalGreeting = "Hi \(nickName ?? fullName)"
+```
+
+As instruções `switch` aceitam qualquer tipo de dado e uma variedade ampla de operações de comparação - elas não são limitadas a números inteiros e testes de igualdade.
+
+```swift
+let vegetable = "red pepper"
+switch vegetable {
+case "celery":
+    print("Adicione umas passas e faça um 'Ants on a Log'")
+case "cucumber", "watercress":
+    print("Isso daria um bom sanduíche")
+case let x where x.hasSuffix("pepper"):
+    print("É um \(x) apimentado?")
```

Comment:
No original, o resultado do switch seria "Is it a spicy red pepper?". Da forma como está, o resultado na versão traduzida seria "É um red pepper apimentado?". Acho que, se traduzir _red pepper_, a idéia original ficaria melhor como "É uma apimentada pimenta vermelha?" - `"É uma apimentada \(x)?"`